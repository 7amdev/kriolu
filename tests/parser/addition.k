2 * -3 / 6;
ka verdadi;
ka nulo;
ka 12;
nulo + verdadi;