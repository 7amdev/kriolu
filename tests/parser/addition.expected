<expression statement>
  ((1.0 + 2.0) + 3.0)
<expression statement>
  ((3.0 + 2.0) + 1.0)
<expression statement>
  ((19.0 + 108.0) + 1017.0)